### PR TITLE
DHFPROD-9565: New entity property selector

### PR DESCRIPTION
**Pull request by iveyMarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- [n/a] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx [hunk 1]
```diff
@@ -458,9 +459,9 @@ const AddEditRelationship: React.FC<Props> = (props) => {
     //menuProps.unshift({value: "None", label: "None", type: "string"});
     if (menuProps) {
       if (!isConceptClassJoinView) {
-        menuProps.unshift({value: "None", label: "None", type: "string"});
+        menuProps.unshift({value: "None", key: "None", label: "None", type: "string"});
       } else {
-        menuProps.unshift({value: ".", label: "instance", type: "string"});
+        // menuProps.unshift({ value: ".", key:".", label: " - instance - ", type: "string" });
```
Comment (littlecherro) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx:464]: there's a comment here, should be removed but is it there for a reason or something?

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx [hunk 2]
```diff
@@ -708,44 +705,76 @@ const AddEditRelationship: React.FC<Props> = (props) => {
...
+  //new property select
+  const [displayPropertyMenu, setDisplayPropertyMenu] = useState(false);
+  const [displayPropertyList, setDisplayPropertyList] = useState(false);
+
+  const handleEntityPropertyList = async (e) => {
+    let name = e.target.value;
+    setSourcePropertyValue(name);
+    if (!displayPropertyList && !displayPropertyMenu) {
+      setSourcePropertyValue("");
+      await setDisplayPropertyList(true);
```
[the remark this answers is not in the document]
Comment (iveyMarklogic) [reply marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.tsx]: @littlecherro both comments were fixed on my last updated.

**Review by prolonMarkLogic — DISMISSED** [review]

**Review by littlecherro — APPROVED** [review]

**Review by MSBrignoni — COMMENTED** [review]
@iveyMarklogic when we have a structured property and another nested with the same name, the component is not showing the "/" properly, this can be tackled in this PR or another.

https://user-images.githubusercontent.com/89084298/196709315-5e5f82db-dca3-4b05-848c-a4f3256835be.mp4

**Comment by rahulvudutala** [comment]
@iveyMarklogic Did we conclude on using dot(.) notation or (/) notation for the context?
Also, the indentation for shipping, zip, fiveDigit doesn't look correct in the attached screenshot. nicknames and shipping should be on the same level. fiveDigit, plusFour and geopoint should be right indented to zip and should be on the same level

Screen Shot 2022-10-19 at 8 35 42 AM

**Review by rahulvudutala — CHANGES_REQUESTED** [review]
@iveyMarklogic Did we conclude on using dot(.) notation or (/) notation for the context?
Also, the indentation for shipping, zip, fiveDigit doesn't look correct in the attached screenshot. nicknames and shipping should be on the same level. fiveDigit, plusFour and geopoint should be right indented to zip and should be on the same level

Screen Shot 2022-10-19 at 8 30 02 AM

**Comment by iveyMarklogic** [comment]
@rahulvudutala and @MSBrignoni the indentation was fixed in my last update
image
image

**Comment by iveyMarklogic** [comment]
@rahulvudutala we conclude on using (/) notation to be consistent with the mapping and also with the expression section
image

**Review by rahulvudutala — APPROVED** [review]

**Review by MSBrignoni — APPROVED** [review]

**Pull request merged**